BD-2114: Update GTM docs w/ optional push setup (#5577)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/web/google_tag_manager.md

Use the Initialization Tag to add the Braze Web SDK to your website.

### Step 1: Push setup (optional)

> Optionally, If you want to be able to send push through the Google Tag Manager, first follow the [push integration]({{site.baseurl}}/developer_guide/platform_integration_guides/web/push_notifications/integration/) guidelines to:

Reviewer: ```suggestion
Optionally, if you want to be able to send push through the Google Tag Manager, first follow the [push integration]({{site.baseurl}}/developer_guide/platform_integration_guides/web/push_notifications/integration/) guidelines to:
```

> 1. Configuring your site's service worker, placing it in the root directory of your site

Reviewer: ```suggestion
1. Configure your site's service worker, placing it in the root directory of your site
```

> 2. Set up browser registration - Once the service worker is configured, your must set the `braze.requestPushPermission()` method either natively in their app or through a custom HTML tag (via the GTM dashboard). You will also need to make that the tag is fired abter the SDK has been initalized.

Reviewer: ```suggestion
2. Set up browser registration - Once the service worker is configured, you must set the `braze.requestPushPermission()` method either natively in their app or through a custom HTML tag (via the GTM dashboard). You will also need to make sure that the tag is fired after the SDK has been initalized.
```
